cmd/kots/cli/install.go

```diff
@@ -166,7 +166,6 @@ func InstallCmd() *cobra.Command {
 				if err := upload.Upload(rootDir, uploadOptions); err != nil {
 					return err
 				}
-				close(stopCh)
```

Comment:
this channel is closed in a defer.  this caused a panic when CTRL+C is pressed at the end of install.